adapta/storage/blob/README.md

```diff
@@ -63,13 +63,64 @@ s3_path = S3Path.from_hdfs_path('s3a://bucket/path/to/my/table')
 
 # init storage client
 s3_client = S3StorageClient(base_client=aws_client)
+s3_client.initialize_session_resource()
 
 # Save data to S3
-
 s3_client.save_data_as_blob(
     data={"data_value": "2"}, blob_path=s3_path, serialization_format=DictJsonSerializationFormat, overwrite=True
 )
 
 # read files from S3
 blobs = s3_client.read_blobs(s3_path, serialization_format=DictJsonSerializationFormat)
 ```
+
+#### Download a single blob:
+```python
+from adapta.security.clients import AwsClient
+from adapta.storage.models.aws import S3Path
+from adapta.storage.blob.s3_storage_client import S3StorageClient
+from adapta.storage.models.format import DictJsonSerializationFormat
+
+# Set up environment variables
+os.environ["PROTEUS__AWS_ACCESS_KEY_ID"] = <aws_access_key_id>
+os.environ["PROTEUS__AWS_SESSION_TOKEN"] = <aws_session_token>
+os.environ["PROTEUS__AWS_SECRET_ACCESS_KEY"] = <aws_secret_access_key>
+os.environ["PROTEUS__AWS_REGION"] = "eu-central-1"
+os.environ["PROTEUS__AWS_ENDPOINT"] = "http://example.com"
+
+# Create client
+credentials = EnvironmentAwsCredentials()
+aws_client = AwsClient(credentials)
+
+# Initialize session
+aws_client.initialize_session()
+
+# Target path for copy_blob
+blob_path = "path/to/blob.file" "# It can be either a 'blob.file' or a 'folder/'
+s3_path = S3Path.from_hdfs_path(blob_path)
+
+# Init storage client
+s3_client = S3StorageClient(base_client=aws_client)
+s3_client.initialize_session_resource()
+
+# Save data to S3
+s3_client.save_data_as_blob(
+    data={"data_value": "very_important_data"}, blob_path=s3_path, serialization_format=DictJsonSerializationFormat, overwrite=True
+)
+
+# List blobs in S3
+blob_list = s3_client.list_blobs(s3_path)
+for blob in blob_list:
+    print(blob)
```

Comment:
We see the table previously stored, it will look like a human friendly table interface:

```
         Character  ... Popularity
0    Frodo Baggins  ...         10
1     Harry Potter  ...          9
2  Sherlock Holmes  ...          8
3       Tony Stark  ...         10
4      Darth Vader  ...          7
```

Should  I mentioned that in the ReadMe?

Reply:
I think I get your point, I've updated the ReadMe file in [8b68ae3](https://github.com/SneaksAndData/adapta/pull/437/commits/8b68ae32118360ff6fc83a043f1ba072c0d82830) to reflect that it displays the blob file details.

So this particular print would show this (for example):
```
{'Key': 'ANCOS@ecco.com/delta_table_test2/0-3636a208-ce85-4edd-9ae5-40c220f5293f-0.parquet', 'LastModified': datetime.datetime(2024, 6, 19, 14, 42, 33, 85000, tzinfo=tzutc()), 'ETag': '"29097d7d2d11d49fed28745a674af776"', 'Size': 2067, 'StorageClass': 'STANDARD', 'Owner': {'DisplayName': 'minio', 'ID': '02d6176db174dc93cb1b899f7c6078f08654445fe8cf1b6ce98d8855f66bdbf4'}}
{'Key': 'ANCOS@ecco.com/delta_table_test2/0-77737968-2543-4078-a13b-e8e967026e43-0.parquet', 'LastModified': datetime.datetime(2024, 6, 24, 9, 27, 59, 630000, tzinfo=tzutc()), 'ETag': '"29097d7d2d11d49fed28745a674af776"', 'Size': 2067, 'StorageClass': 'STANDARD', 'Owner': {'DisplayName': 'minio', 'ID': '02d6176db174dc93cb1b899f7c6078f08654445fe8cf1b6ce98d8855f66bdbf4'}}
{'Key': 'ANCOS@ecco.com/delta_table_test2/0-81ae5254-0fce-478f-af5f-6d134612b144-0.parquet', 'LastModified': datetime.datetime(2024, 6, 24, 9, 21, 15, 180000, tzinfo=tzutc()), 'ETag': '"29097d7d2d11d49fed28745a674af776"', 'Size': 2067, 'StorageClass': 'STANDARD', 'Owner': {'DisplayName': 'minio', 'ID': '02d6176db174dc93cb1b899f7c6078f08654445fe8cf1b6ce98d8855f66bdbf4'}}
```